hls4ml/backends/vivado/vivado_backend.py

```diff
@@ -649,3 +649,61 @@ def init_garnet(self, layer):
     @layer_optimizer(GarNetStack)
     def init_garnet_stack(self, layer):
         self.init_garnet(layer)
+
+    def generate_pointwise_conv1d_fn(self, layer_idx, reuse_factor=1):
```

Comment:
Does it make sense to put this instead in `pointwise_codegetn.py`?

Reply:
Yes, done.